docs/OnTrack/Jupyter Notebook/Docker-Containers-SRS.md

```diff
@@ -0,0 +1,133 @@
+# Jupyter Notebook/Word Document Docker Containers Software Requirement Specification (SRS) Document
+
+---
+
+## 1. Introduction
+
+---
+
+### 1.1 Purpose
+
+- Currently, when using the OnTrack system, the user would need to convert a Jupyter Notebook file,
+  or a Word document file to PDF before uploading to OnTrack. The purpose of the Docker containers
+  is to allow the conversion of both Jupyter Notebook and Word document file types to a PDF file
+  type, in the process of submitting them to OnTrack. The user will submit either a Jupyter Notebook
+  file or a Word Document file, and depending on which file type is submitted, one container will
+  provide the conversion function for the Jupyter Notebook, and the other container will provide the
+  conversion function for the Word document. Both containers will output a PDF file.
+
+### 1.2 Intended Audience
+
+- The intended audience of this software is all users of the OnTrack system (students and teachers).
+  This software will allow all users to submit Jupyter Notebook and Word Document files to OnTrack
+  directly, instead of having to first manually convert the file to PDF and then submit that to
+  OnTrack. The software will also allow users to view there converted file for review or marking.
```

Comment:
```suggestion
- The intended audience of this feature is all users of the OnTrack system (students and teachers).
  This feature will allow all users to submit Jupyter Notebook and Word Document files to OnTrack
  directly, instead of having to first manually convert the file to PDF and then submit that to
  OnTrack. The feature will also allow users to view their converted file for review or marking.
```